Neos.Eel/Classes/Helper/DateHelper.php

```diff
@@ -57,6 +71,31 @@ public function format($date, $format)
     }
 
     /**
+     * Format a date to a string with a given cldr format
+     *
+     * @param \DateTime $date
+     * @param string $cldrFormat Format string in CLDR format (see http://cldr.unicode.org/translation/date-time)
+     * @param null|string $locale String locale - example (de|en|ru_RU)
+     *
```

Comment:
Well, I prefer test coverage tbh. Basically that just shows that this method uses too much delegation, esp. the `$this->localizationService->getConfiguration()->getCurrentLocale();` for example violates the "Law of Demeter". So this should possibly be refactored, either by adding a method to localizationService or by adding a (protected) method in this helper, that fetches the current locale.